packages/fondue/package.json

```diff
@@ -59,6 +59,7 @@
         "test:file": "cypress run --component --spec",
         "typecheck": "tsc --noEmit",
         "storybook": "storybook dev -p 6006",
+        "storybook:background": "storybook dev --no-open -p 6006",
```

Comment:
```suggestion
        "storybook:watch": "storybook dev --no-open -p 6006",
```

Reply:
so this command is only required when spinning up  the _root_ Storybook right?

Then we don't need this script at all, the root command can just pipe through the flag:

Can you test this?

`"storybook": "pnpm -r storybook -- --no-open"`

I am not sure, but with pnpm it might also work without the explicit passthrough `--`

Reply:
script adjusted